app.py

```diff
@@ -5,42 +5,59 @@
 from dotenv import load_dotenv
 load_dotenv()
 
-import code
 
-
-# Init app
 app = Flask(__name__)
 
 
-# Get solar system data from NREL API
 @app.route('/results', methods=['POST'])
-def get_solar_system():
+def get_solar_data():
     results = NREL(request.json)
-    payload = results.get()
-    payload = json.loads(payload.text)
-    return jsonify(payload)
+    system_payload = results.get_system_info()
+    value_payload = results.get_kwh_rate_info()
+    calculated_value = results.calculate_value(value_payload)
+    system_and_value_payload = {**system_payload, **calculated_value}
+    return jsonify(system_and_value_payload)
```

Comment:
Awesome work here. Looks like you got both the system and the value returning as JSON to the FE?